data/_models/shared/Resources.csv

```diff
@@ -35,8 +35,8 @@ Resources,,record trigger,text,,,,,,,,,"What triggers the creation of a record i
 Resources,,unit of observation,text,,,,,,,,,"Based on the record trigger or promt, what is the unit(s) of observation of a record (e.g. person, prescription)?","DataCatalogueFlat,RWEStaging"
 Resources,,subpopulations,refback,,,,Subpopulations,,resource,,,"List of cohorts or subpopulations defined in this resource, if applicable","DataCatalogueFlat,CohortsStaging,UMCGCohortsStaging,NetworksStaging,UMCUCohorts"
 Resources,,collection events,refback,,,,Collection events,,resource,,dcterms:hasPart,"List of data/sample collection events defined for this collection, if applicable","DataCatalogueFlat,CohortsStaging,UMCGCohortsStaging,NetworksStaging,UMCUCohorts"
-Resources,,resources,ref_array,,,,Resources,,,,dct:dataset,List of other resources included in this resource,"DataCatalogueFlat,EMA"
-Resources,,part of resources,refback,,,,Resources,,resources,,dct:dataset,List of other collections included in this resource,"DataCatalogueFlat,EMA"
+Resources,,resources,ref_array,,,,Resources,,,,dcat:dataset,List of other resources included in this resource,"DataCatalogueFlat,EMA"
```

Comment:
heb je gecheckt welke prefix we nu gebruiken op RDF api nivo?

Reply:
dct and dcat are different things, dct is dublin core, dcat is dcat. for dct the prefix in the rdf is dcterm, so I will also check that.